AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -20,7 +21,7 @@ @interface MSAuthTokenContext ()
 /**
  * The last value of user account id.
  */
-@property(nullable, nonatomic, copy) NSString *homeAccountId;
+@property(nullable, atomic, copy) NSString *accountId;
```

Comment:
Let's not rename if homeAccountId and accountId are different strings.

The Android client keeps the 2 terms distinct (accountId is only the guid, while homeaccountid includes tenant and policy). In Android MSAL get account requires homeAccountId while we expose only accountId in signIn method result.